casa/System/Aipsrc.cc

```diff
@@ -549,15 +552,14 @@ uInt Aipsrc::genRestore(Vector<String> &namlst, Vector<String> &vallst,
   Block<String> nl;
   Block<String> vl;
   Int nkw = Aipsrc::genParse(nl, vl, ef, fileList);
-  Block<String> nla;
-  Block<String> vla;
-  nla.resize(0);
-  vla.resize(0);
+  std::vector<String> nla;
+  std::vector<String> vla;
   uInt n;
   for (Int i=nkw-1; i>=0; i--) {	// reverse order to do aipsrc like
     if (!nl[i].contains('*')) {		// no wild cards
       n = Aipsrc::registerRC(nl[i], nla);
-      vla.resize(n);
+      if(n > vla.size())
+        vla.resize(n);
```

Comment:
Since `registerRC` already resizes the list, can't you simply remove this code?
```suggestion
```

Reply:
Also as before, but now `nla` vs `vla`.